frontend/src/pages/HistoryFileCommitPage.tsx

```diff
@@ -3,31 +3,102 @@ import { useDataSettingContext } from "../context/DataSettingContext";
 import "assets/styles/evolutionFileCommit.scss";
 import DateAndFileInput from "components/DateAndFileInput";
 import MotionChartDisplay from "components/MotionChartDisplay";
-import { FileHistoryCommit } from "models/FileHistoryCommit"
+import { FileHistoryCommit } from "models/FileHistoryCommit";
 
-import { fileHistoryByDate } from "api";
+import { fileHistoryByDate, getFileData } from "api";
```

Comment:
getFileData is not used, why is it imported?

Reply:
It's done